docs/howtos/linkml-project-cookiecutter.md

```diff
@@ -0,0 +1,172 @@
+# LinkML Project Cookiecutter
+
+A [Cookiecutter](https://cookiecutter.readthedocs.io/en/stable/) template for projects using [`Linkml`](https://github.com/linkml/linkml).
+
+
+## Standard Protocol
```

Comment:
Will we delete the docs in the README of the cookiecutter repo, so we don't have to sync docs?

I am not convinced this route involving creating a poetry project just for cruft is the best long term approach, but so long as we have one SoT to update when we have a better way we should move forward!

Reply:
I had the same thought about cp/pasted doc.  I think its good to have it close to the repo itself as folks will be cloning from that location, but it also seemed prudent to have it here in case someone was looking for "howto" get started.  I condensed the doc here significantly into a "quickstart" guide that isn't redundant with the README in the cookie-cutter and is a bit more generic in its instructions.